src/Pelagos/Bundle/AppBundle/Resources/public/js/data-discovery.js

```diff
@@ -237,40 +236,19 @@ function createRow(data, row)
 
     if (!data["doi"]["doi"]) {
         $(rowContent).find("#container-dataset-doi").hide();
-    } else if (activeTabIndex === 2) {
-        // Remove the hyperlink, but keep text for this tab.
-        $(rowContent).find("#dataset-doi").replaceWith(data["doi"]["doi"]);
     } else {
-        $(rowContent).find("#dataset-doi").attr("href", $(rowContent).find("#dataset-doi").attr("href").replace("placeholder-doi", data["doi"]["doi"]));
-        $(rowContent).find("#dataset-doi").text(data["doi"]["doi"]);
+        $(rowContent).find("#dataset-doi").replaceWith(data["doi"]["doi"]);
     }
 
-    $(rowContent).find("#dataset-udi").attr("href", $(rowContent).find("#dataset-udi").attr("href").replace("placeholder-udi", data["udi"]));
-    $(rowContent).find("#dataset-udi").text(data["udi"]);
+    $(rowContent).find("#dataset-udi").replaceWith(data["udi"]);
 
 
 
     //this shows the details dataset on Show Details button
-    $(".details_link", rowContent).on("click", function(){
```

Comment:
You should probably retain the way this is called, because in the end we are returning rowContent.
row is an earlier element.